<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShutDownDlg</class>
 <widget class="QWidget" name="ShutDownDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShutDownDlg</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>243</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancerBtn">
   <property name="geometry">
    <rect>
     <x>477</x>
     <y>243</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBtn_exitProgram">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>301</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBtn_switchUsers">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>301</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>切换用户</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
